so that you're always using the LATEST...

nickboldt opened the pull request:
so that you're always using the LATEST version of the central/earlyaccess site, use TARGET_PLATFORM_CENTRAL_MAX from parent pom instead of hardcoding an old version here

alexeykazakov commented:
> Request/plugins/org.jboss.tools.jst.web/pom.xml
> [ERROR] Internal error: java.lang.RuntimeException: Invalid repository URL: http://download.jboss.org/jbosstools/targetplatforms/jbtearlyaccesstarget/${TARGET_PLATFORM_CENTRAL_MAX}/REPO/: Illegal character in path at index 75: http://download.jboss.org/jbosstools/targetplatforms/jbtearlyaccesstarget/${TARGET_PLATFORM_CENTRAL_MAX}/REPO/ -> [Help 1]

nickboldt commented:
Damn. OK, so we can't nest variables like this. Will try another approach.

nickboldt commented:
Error was actually caused by typo in parent pom. Fixed upstream in parent pom and now this PR works too. Added more simplifications you could do if you want them.

alexeykazakov commented:
testPR

alexeykazakov commented:
@rawagner please take a look at failing Integration tests check

nickboldt commented:
added -U -e to the pr test job so we get latest parent pom, and when things break you get a full stack track instead of "error 13".

nickboldt commented:
testPR

alexeykazakov commented:
@nickboldt please fix the conflict if you think it makes sense to apply this PR

nickboldt commented:
Conflict fixed. Merge if you want this.

nickboldt commented:
replaced by #578 (which contains this change too)
